webapp/layouts/login.vue

```diff
@@ -0,0 +1,8 @@
+<template>
+  <div class="container mt-4">
+    <Nuxt />
+  </div>
+</template>
+
+<style src="bootstrap/dist/css/bootstrap.css">
+</style>
```

Comment:
Create a menu component with a <nuxt-link> to /login if the user is not logged in. If the user is logged in, it shows a logout button. You might want to put this this menu component in your layouts/default.vue. Furthermore, you might want to use Vuex for a globally accessible isAuthenticated getter method. There is a nuxt integration.